### PR TITLE
Fix test executable

### PR DESCRIPTION
**Pull request by stefan-caraiman**
Adds different executables for 32bit and 64bit versions of Windows,
since Windows Nano has been stripped of syswow64 and it isn't able
to run 32bit executable versions.

### CODE REVIEW

File: argus/recipes/cloud/windows.py [hunk 1]
```diff
@@ -354,7 +354,14 @@ def pre_sysprep(self):
         super(CloudbaseinitScriptRecipe, self).pre_sysprep()
         LOG.info("Doing last step before sysprepping.")
 
-        resource_location = "windows/test_exe.exe"
+        cmd = '$ENV:PROCESSOR_ARCHITECTURE'
+        stdout = self._backend.remote_client.run_command_verbose(cmd)
+        architecture = stdout.strip()
+        if architecture == 'AMD64':
+            resource_location = "windows/test_exe64.exe"
```
Comment (ader1990) [inline argus/recipes/cloud/windows.py]: use just one resource_location definition with a formatted string
smth like "arch %s".format(if (arch==amd64)? 64:32)

**Review by ader1990 — CHANGES_REQUESTED** [review]

**Review by ader1990 — APPROVED** [review]
lgtm

**Comment by mateimicu** [comment]
lgtm

**Review by ader1990 — APPROVED** [review]
lgtm

**Pull request merged**